### ISSUE TITLE
Strange behaviour in function _p()

### ISSUE THREAD

**Issue by xavi-mat** [renpy/renpy#2673]
Strange behaviour in translation function `_p()` when using quotes inside the text.
This code:
```py
define gui.about = _p("""
En "Atrapados en Órbita" tomas el papel...

...se pueden configurar en "Opciones".
""")
```
Produces this output:
![screenshot0006](https://user-images.githubusercontent.com/1102677/108374061-d5f37200-7200-11eb-8f3e-17a045157410.png)

There are no line breaks. Also, look at the first quotes: with the `u"Atrapados...` , and the last word (`"Opciones"`), correct without the `u`.

Escaping the internal quotes, this second code: 
```py
define gui.about = _p("""
En \"Atrapados en Órbita\" tomas el papel...

...se pueden configurar en \"Opciones\".
""")
```
Produces this output:
![screenshot0005](https://user-images.githubusercontent.com/1102677/108374074-da1f8f80-7200-11eb-9b50-f3abb0cd9d25.png)

We have our line break back, but the strange `\u` is still there... in `"Atrapados`, not in `"Opciones"`

**Comment by Gouvernathor** [comment]
After checking, in your first test, the _p function receives the string as `En "Atrapados en Órbita" tomas el papel......se pueden configurar en "Opciones".`, so the glitch doesn't happen inside it, but rather before it's even called, when its arguments are evaluated.
Weird, and I don't know which part of the code might be responsible.

**Comment by renpytom** [comment]
Should be fixed by de35ffe27 , though it might require a force recompile to fix things.
